helm/ngrok-operator/CHANGELOG.md

```diff
@@ -1,20 +1,72 @@
+- Implement BindingEndpoint polling by @hjkatz in [#458](https://github.com/ngrok/ngrok-operator/pull/458)
+- Implement EndpointBinding -> Services creation by @hjkatz in [#459](https://github.com/ngrok/ngrok-operator/pull/459)
+- Implement port allocation by @hjkatz in [#460](https://github.com/ngrok/ngrok-operator/pull/460)
+- Bindings forwarder by @jonstacks in [#465](https://github.com/ngrok/ngrok-operator/pull/465)
+- Add endpoint status to EndpointBinding kubectl output by @hjkatz in [#464](https://github.com/ngrok/ngrok-operator/pull/464)
+- Ensure endpoint poller does not start until k8sop is regestered with API by @hjkatz in [#470](https://github.com/ngrok/ngrok-operator/pull/470)
+- Rename EndpointBinding to BoundEndpoint by @hjkatz in [#475](https://github.com/ngrok/ngrok-operator/pull/475)
+- Implement Target Metadata by @hjkatz in [#477](https://github.com/ngrok/ngrok-operator/pull/477)
+- Bindings forwarder implementation by @jonstacks in [#476](https://github.com/ngrok/ngrok-operator/pull/476)
+
+#### Helm values.yaml schema validation
+
+The `ngrok-operator` helm chart now includes a `schema.json` file that can be used to validate the `values.yaml` file.
+
+- Generate and commit schema.json file by @alex-bezek in [#472](https://github.com/ngrok/ngrok-operator/pull/472)
+
 ### Changed
 
-- Deprecate `.Values.metaData` in favor of `.Values.ngrokMetadata` for clarity
-- Deprecate `.Values.ingressClass` in favor of `.Values.ingress.ingressClass` for feature set namespacing
-- Deprecate `.Values.useExperimentalGatewayApi` in favor of `.Values.gateway.enabled` for feature set namespacing
-- Deprecate `.Values.watchNamespace` in favor of `.Values.ingress.watchNamespace` for feature set namespacing
-- Deprecate `.Values.controllerName` in favor of `.Values.ingress.controllerName` for feature set namespacing
+#### Traffic Policy
+
+Updates `TrafficPolicy` CRD and inline policy to support new phase-based names as well as the new `TrafficPolicy` API.
+
+- update traffic policy for phase-based naming by @TheConcierge in [#456](https://github.com/ngrok/ngrok-operator/pull/456)
+
+#### Values.yaml file deprecations
+
+- Rename .Values.metaData -> .Values.ngrokMetadata by @hjkatz in [#434](https://github.com/ngrok/ngrok-operator/pull/434).
+  - Deprecate `.Values.metaData` in favor of `.Values.ngrokMetadata` for clarity
+  - Deprecate `.Values.ingressClass` in favor of `.Values.ingress.ingressClass` for feature set namespacing
+  - Deprecate `.Values.useExperimentalGatewayApi` in favor of `.Values.gateway.enabled` for feature set namespacing
+  - Deprecate `.Values.watchNamespace` in favor of `.Values.ingress.watchNamespace` for feature set namespacing
+  - Deprecate `.Values.controllerName` in favor of `.Values.ingress.controllerName` for feature set namespacing
+
+### New Contributors
+- @TheConcierge made their first contribution in [#456](https://github.com/ngrok/ngrok-operator/pull/456)
+
```

Comment:
This isn't in your change but is just below and related tot he rename. Down below it has the deprecation announcement and has a link to github discussions for the full announcement. 

> See Full Announcement: https://github.com/ngrok/ngrok-operator/discussions

I don't see anything here, is this the right spot and we are just holding the announcement?

Reply:
Fixed